lib/controller/api_confluence.es6.js

```diff
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
 'use strict';
 
+/* global M */
```

Comment:
Perhaps briefly mention what is `M` (I suppose it's an Angular JS thing? But the keyword is too short to search for.)

Reply:
Added a comment: M is Materialize